include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -198,31 +191,11 @@ namespace aspect
 
 
         /**
-         * Retrieve the rot_para_to_para_angle variable.
-         */
-        double
-        get_rot_para_to_para_angle() const;
-
-
-        /**
-         * Retrieve the para_to_rect_angle variable.
+         * Retrieve the corners used to create the
+         * ellispoid.
          */
-        double
-        get_para_to_rect_angle() const;
-
-
-        /**
-         * Retrieve the rotation_longitude variable.
-         */
-        double
-        get_rotation_longitude() const;
-
-
-        /**
-         * Retrieve the rotation_latitude variable.
-         */
-        double
-        get_rotation_latitude() const;
+        std::vector<std::vector<double> >
+        get_corners() const;
```

Comment:
Here too, I think you'd be better off returning a `const` reference, rather than a copy.

Can you explain in the documentation of the function how "corners" are defined, and how that corresponds to the elements of the vector?